documentation/staging/content/userguide/base-images/patch-images.md

```diff
@@ -126,17 +124,7 @@ The `rebase` command does the following:
 
 Using `rebase`, the new image can be created in one of two ways:
 
-- As a new WebLogic image from a base OS image (similar to the `create` command; recommended).
-
-  To activate:
-  - Set `--tag` to the name of the final new image.
-  - Set `--sourceImage` to the WebLogic image that contains the WebLogic configuration.
-  - Set additional fields (such as the WebLogic and JDK locations),
-    similar to those used by `create`.
-    See [Create a custom base image](#create-a-custom-base-image).
-  - Do _not_ set `--targetImage`.  (When
-    you don't specify a `--targetImage`, `rebase` will use
-    the same options and defaults as `create`.)
+- As a new WebLogic image from a base OS image (similar to the `create` command; recommended). 
 
 - Or, as a new layer on an existing
```

Comment:
Or, use as base image the WebLogic Server CPU images from OCR that doesn't already have a domain home.

In the example bellow you want to use 
--targetImage  container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol8

Reply:
Done.